go/vt/vtorc/inst/keyspace_dao.go

```diff
@@ -30,6 +30,10 @@ var ErrKeyspaceNotFound = errors.New("keyspace not found")
 
 // ReadKeyspace reads the vitess keyspace record.
 func ReadKeyspace(keyspaceName string) (*topo.KeyspaceInfo, error) {
+	if err := topo.ValidateKeyspaceName(keyspaceName); err != nil {
+		return nil, err
+	}
```

Comment:
In the upstream PR we documented this as a breaking change. And here I can see that it could be. Do we really want to backport this? If so, don't we want the same release notes summary update?

Reply:
yes, we want the backport, i'll update the notes!